pkg/templates/kubeadm/v1beta1/kubeadm.go

```diff
@@ -160,7 +159,7 @@ func NewConfig(s *state.State, host kubeoneapi.HostConfig) ([]runtime.Object, er
 		CgroupDriver:       "systemd",
 		ReadOnlyPort:       0,
 		RotateCertificates: true,
-		ClusterDNS:         []string{nodelocaldns.VirtualIP},
+		ClusterDNS:         []string{"169.254.20.10"},
```

Comment:
This should be moved to some sort of common "resources" package

Reply:
I've added the `resources` package as suggested. PTAL.